extensions/amp-date-countdown/amp-date-countdown.md

```diff
@@ -91,6 +93,10 @@ This table provides examples of formatted values specified in a Mustache templat
 | {h} {hours} {m} {minutes}                       | 240 hours 10 minutes                 | `biggest-unit='hours'` |
 | {d} {days} {h} {hours} {m} {minutes}            | 50 天 5 小时 10 分钟                 | `locale='zh-cn'`       |
 
+### Migrating from 0.1
+
+Unlike `0.1`, the experimental `1.0` version of `amp-date-countdown` does not currently support the `data-count-up` attribute which enables the component to count in the opposite direction.
```

Comment:
```suggestion
The experimental `1.0` version of `amp-date-countdown` does not support the `data-count-up` attribute. This means the `1.0` component is unable to count in the opposite direction.
```

Reply:
Thanks, I used this suggestion in the latest revision.